experiments/modern/src/runtime/MakiObject.js

```diff
@@ -1,10 +1,22 @@
 const Emitter = require("../Emitter");
 
 class MakiObject {
-  constructor() {
+  constructor(node, parent) {
+    this.xmlNode = node;
+    this.parent = parent;
+    this.children = [];
+    this.hooks = {};
     this._emitter = new Emitter();
   }
 
+  addChild(child) {
+    this.children.push(child);
+  }
```

Comment:
Is this used?

Reply:
not yet, but will be used by `init` later on..I guess we could just use `addChildren([child])`, and have 1 method, but it was easy enough to write :D